### PR TITLE
Login with staging url added

### PR DESCRIPTION
**Pull request by ashifkhn**
Date: `change date here`

Developer Name: `Ashif`

----

## Issue Ticket Number

-   Add the relevant issue ticket number here (e.g., Closes #<issue_number>).

## Description

Provide a concise description of the changes made in this PR.
Staging api for login has been added, app can be used in staging by chnaging the base url in the api files.

https://github.com/Real-Dev-Squad/mobile-app/assets/54736284/7aaae072-a057-420a-a313-9534339ac523

**Documentation Updated?**

-   [ ] Yes
-   [x] No

_If your feature adds a new API, then documentation related to the feature like API-Contracts, and Data-models  must be updated._

**Breaking Changes**

-   [ ] Yes
-   [x] No

_If your feature introduces breaking changes or if something is missing, please mention the related issue tickets._

**Development Tested?**

-   [x] Yes
-   [ ] No

_Confirm whether the changes have been tested locally during development._

**Tested in Staging?**

-   [x] Yes
-   [ ] No

_Indicate whether the changes have been tested in the staging environment._

**Under Feature Flag**

-   [x] Yes
-   [ ] No

_Specify if the changes are currently under a feature flag._

**Database Changes**

-   [ ] Yes
-   [x] No

_Indicate whether the changes include modifications to the database._

### Screenshots

Attach any relevant screenshots, such as test coverage reports, before and after images, or other visual aids.

## Test Coverage

Provide information on the test coverage for the changes made in this PR. Include details on the types of tests conducted and their outcomes.

## Additional Notes

Include any additional notes, considerations, or explanations that might be helpful for reviewers.

### CODE REVIEW

File: src/components/DisplayContribution.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ import { profileScreenStyles } from '../screens/ProfileScreen/styles';
 
 const DisplayContribution = ({ tasks }) => {
   return (
-    <View style={{ padding: 5, width: "100%", alignItems: 'center' }}>
+    <View style={{ padding: 5, width: '100%', alignItems: 'center' }}>
```
Comment (shreya-mishra) [inline src/components/DisplayContribution.tsx]: why single quote change?

File: src/components/DisplayContribution.tsx [hunk 2]
```diff
@@ -166,7 +166,7 @@ const DisplayContribution = ({ tasks }) => {
           </View>
         ))
       ) : (
-        <Text style={{color: '#000'}}>No Tasks Yet!</Text>
+        <Text style={{ color: '#000' }}>No Tasks Yet!</Text>
```
Comment (shreya-mishra) [inline src/components/DisplayContribution.tsx]: +1

File: src/constants/apiConstant/AuthApi.ts [hunk 3]
```diff
@@ -1,6 +1,9 @@
+// const basrUrl = 'https://api.realdevsquad.com'; //production
+
 const AuthApis = {
   USER_DETAIL: 'https://api.realdevsquad.com/users/userId/',
   QR_AUTH_API: 'https://api.realdevsquad.com/auth/qr-code-auth',
+  QR_AUTH_API_STAGING: 'https://staging-api.realdevsquad.com/auth/qr-code-auth',
```
Comment (shreya-mishra) [inline src/constants/apiConstant/AuthApi.ts]: move it to baseConstant file

File: src/components/DisplayContribution.tsx (hunk 1 above)
Comment (ashifkhn) [reply src/components/DisplayContribution.tsx]: Happend because of prettier, Ill disable.

File: src/components/DisplayContribution.tsx (hunk 2 above)
Comment (ashifkhn) [reply src/components/DisplayContribution.tsx]: Happend because of prettier, Ill disable.

File: src/constants/apiConstant/AuthApi.ts (hunk 3 above)
Comment (ashifkhn) [reply src/constants/apiConstant/AuthApi.ts]: we dont have a file with this name.

**Review by iamitprakash — APPROVED** [review]

**Pull request merged**